ckanext-offgridtheme/ckanext/offgridtheme/static/scripts/maps.js

```diff
@@ -0,0 +1,52 @@
+mapboxgl.accessToken = 'pk.eyJ1IjoiZG1vcmlhcnR5IiwiYSI6Ikd3T29EOWMifQ.-DKJ4ernht84AZmc6Bk51Q';
+var map_1 = new mapboxgl.Map({
+    container: 'ssa-aicd-electricity-map',
+    style: '/data/map-style.json',
+    center: [60, 2.867],
+    zoom: 1.5
+});
+
+map_1.on('load', function () {
+  map_1.addLayer({
+    "id": 'data-layer-kano',
+    "type": 'line',
+    "source": {
+      "type": "geojson",
+      "data": "/data/aicd-eletricity-clean.geojson"
```

Comment:
We should use the Vector Tiles that we're already serving from S3 for both these datasets.
You can find the tilejson on the metadata of each dataset.
You can check https://github.com/developmentseed/ckanext-mvt/blob/develop/ckanext/mvt/templates/mvtview.html  on how we're including the tilejson in the other previews.